pkg/addons/addons.go

```diff
@@ -150,6 +150,12 @@ kubectl get secret $SECRET --namespace headlamp --template=\{\{.data.token\}\} |
 minikube{{.profileArg}} addons enable metrics-server	
 
 `, out.V{"profileArg": tipProfileArg})
+	case "yakd":
+		out.Styled(style.Tip, `To access YAKD - Kubernetes Dashboard, wait for Pod to be ready and run the following command:
+
+	minikube service yakd-dashboard -n yakd-dashboard
```

Comment:
```suggestion
	minikube{{.profileArg}} service yakd-dashboard -n yakd-dashboard
```

I'd recommend putting `{{.profileArg}}` there because if the user enables the addon on a cluster with a custom profile your current command will take place against the default profile.

Example:
```
$ minikube start -p p2 --addons yakd
...
To access YAKD - Kubernetes Dashboard, wait for Pod to be ready and run the following command:

minikube service yakd-dashboard -n yakd-dashboard

$ minikube service yakd-dashboard -n yakd-dashboard
🤷  Profile "minikube" not found. Run "minikube profile list" to view all profiles.
👉  To start a cluster, run: "minikube start"
```

Compared to:
```
$ minikube start -p p2 --addons yakd
...
To access YAKD - Kubernetes Dashboard, wait for Pod to be ready and run the following command:

minikube -p p2 service yakd-dashboard -n yakd-dashboard
```